auctions/models.py

```diff
@@ -28,6 +28,8 @@
                          "client_secret": settings.PAYPAL_CLIENT_SECRET,
                          })
 
+PAYPAL_PAYOUT_RECIPIENT = settings.PAYPAL_PAYOUT_RECIPIENT
```

Comment:
Thought the settings assignments should be upfront.  Especially if we use this on other models in the future.